### DISCUSSION TITLE
Does netmiko support lenovo ne1032 switch

### DISCUSSION THREAD

**Discussion by unustayyab123** [ktbyers/netmiko#3114] [Q&A]
Hey guys..

We have some Lenovo ThinkSystem NE1032 RackSwitch, does the netmiko provides support for this switch
i am trying to use netmiko ssh connection to deploy config...

**Comment by ktbyers** [comment]
Probably not. Unless it is fundamentally just a different platform that already exists (i.e. that is just a rebranded version of something else).

Platforms are listed here:

https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md
